content-guidelines/optimize-product-page.md

```diff
@@ -17,28 +17,36 @@ When you build an extension, you make the web better. To get your great idea out
 
 #### What’s Great Content and Design?
 
-Featuring Madhava Enros, Senior Director of Firefox User Experience, and Dietrich Ayala, extension developer.
-
 [Watch the video](a0_OsLGI0k4 'Open Popup Video')
```

Comment:
This might have something to do with the component, but the right-alignment of the video doesn't look great.